x-pack/test/reporting_api_integration/reporting_and_security/usage/api_counters.ts

```diff
@@ -35,10 +36,11 @@ export default function ({ getService }: FtrProviderContext) {
     describe('server', function () {
       this.tags('skipCloud');
       it('configuration settings of the tests_server', async () => {
-        const usage = await usageAPI.getUsageStats();
```

Comment:
I wonder if this will help resolve https://github.com/elastic/kibana/issues/134517

Reply:
I tried unskipping it, but it was still flaky. I can try to debug it on a follow up PR. I wouldn't like to delay this one further because it's blocking a community PR 🧡